src/ompl/geometric/planners/rrt/src/RRTstar.cpp

```diff
@@ -230,6 +277,7 @@ ompl::base::PlannerStatus ompl::geometric::RRTstar::solve(const base::PlannerTer
         }
 
         // Check if the motion between the nearest state and the state to add is valid
+        ++collisionChecks_;
```

Comment:
It is the number of calls to `SpaceInformation::checkMotion`.
The intent was as a planner progress property, as it can be interesting to see how many collision checks a planner is making. I thought it was useful, if your opinion differs, that's fine just let me know. :smile: